db/data/queries/createDatabase.sql

```diff
@@ -1 +1,2 @@
 CREATE DATABASE `quickOrder`;
+ALTER DATABASE `quickOrder` COLLATE = `utf8_general_ci`;
```

Comment:
Already fixed the problem in my pull request https://github.com/konnovdev/food-order-backend/pull/20
So you can remove this commit

Reply:
- [ ]  remove commit

Reply:
I removed it